spec/client_spec.rb

```diff
@@ -209,7 +209,7 @@
       end
 
       context 'with non-serializable data' do
-        let(:data) { [:foo, 'bar'] }
```

Comment:
Right... In that case, it looks like we'll need to do the validation manually: Oj 3, even in `:strict` mode, converts symbols to strings rather then raising an error.